webcompat/templates/home-page/form.html

```diff
@@ -89,41 +89,38 @@ <h1 class="headline-1">Report Site Issue</h1>
       {% endfor %}
     </div>
     <div class="form-upload form-element js-image-upload js-Form-group">
-      <label class="wc-UploadForm wc-UploadForm--homePage js-image-upload-label" for="image">
-        {{ form.image(class_='form-input input-upload wc-ButtonUpload', accept='.jpe,.jpg,.jpeg,.gif,.png,.bmp') }}
+        {{ form.image(class_='form-input input-upload', accept='.jpe,.jpg,.jpeg,.gif,.png,.bmp', id='image') }}
         {% if form.image.errors %}
           {% for error in form.image.errors %}
-          <span class="wc-Form-helpMessage--imageUpload">{{ error }}</span>
+          <span class="upload-error">{{ error }}</span>
           {% endfor %}
         {% endif %}
-        <div class="wc-UploadForm-wrapper">
-          <label class="form-label label-upload wc-UploadForm-label">
-            <svg class="icon icon-big wc-UploadForm-icon" role="presentation">
-              <use class="icon-upload" href="#svg-upload-2">Upload</use>
-            </svg>
-            Upload screenshot
-          </label>
-        </div>
+        <label class="form-label label-upload" for="image">
+          <svg class="icon icon-big wc-UploadForm-icon" role="presentation">
```

Comment:
I don't think `wc-UploadForm-icon` is used anymore, so it could be removed later.

Reply:
Pretty much all `wc-`  styles can be removed.

Reply:
Just grep the tests and update them when you do that -- the odds are pretty good tests are relying on them. :)